service/awsconfig/cluster_framework.go

```diff
@@ -406,6 +408,33 @@ func newClusterResourceRouter(config ClusterFrameworkConfig) (*framework.Resourc
 		}
 	}
 
+	var resourceSetV9 *framework.ResourceSet
```

Comment:
There is also `service/awsconfig/drainer_framework.go` which has to be updated as well.